std_msgs/Int8 nav_waypoint_num
std_msgs/Float32 north_pole_diff
std_msgs/Float32 nav_distance_threshold
std_msgs/Float32 nav_distance_gap
